jobserv/api/worker.py

```diff
@@ -103,7 +105,18 @@ def worker_get(name):
         w.ping(**request.args)
 
     runners = int(request.args.get("available_runners", "0"))
-    if runners > 0 and w.available:
+    disk_free_bytes = int(request.args.get("disk_free", "0"))
```

Comment:
When do we receive this information? Are there possibilities that multiple runs will be planned because they all receive the disk free above the 30 Gb threshold, but the combined runs will not be possible in the 30Gb space?

Reply:
the worker includes it every time it checks in. workers only do a run at a time in practice - but we have to assume this threshold is the minimum required to handle the total number of runs for a worker.